mmdet/models/backbones/efficientnet.py

```diff
@@ -0,0 +1,409 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import copy
```

Comment:
Is this file still necessary? Should we use the efficientnet backbone in mmcls?

Reply:
It needs to change several places to use mmcls efficientnet backbone in mmdet.

Reply:
If it is necessary, we should add comments to indicates the difference and explain the necessity.

Reply:
Done.

Reply:
Done.